FeatureToggle.Domain/Entity/FeatureManagementSchema/Log.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace FeatureToggle.Domain.Entity.FeatureManagementSchema
+{
+    public class Log
+    {
+        public int Id { get; private set; }
+        //public User User { get; private set; }
+        public string UserId { get; private set; }        
+        public int FeatureId { get; private set; }
+        public int? BusinessId { get; private set; }
+        public DateTime Time { get; private set; }
+        public Actions Action { get; private set; }
+
+        public enum Actions
```

Comment:
Move this to a separate folder called ```FeatureStatus```

Reply:
Action is supposed to denote what the user did, 
so "**UserAction**" would be more fitting right?